src/test-reader/mocha-reader/index.js

```diff
@@ -95,6 +99,94 @@ function applyOnly(rootSuite, eventBus) {
     });
 }
 
+function addLocationToRunnables(inBus, config, runnableOpts) {
+    if (!runnableOpts || !runnableOpts.saveLocations) {
+        return;
+    }
+
+    const sourceMapSupport = tryToRequireSourceMapSupport();
```

Comment:
Check if `source-map-support` from `ts-node` exists in user node_modules. It is used to correctly determine the position of the declaration suite and test (really helpful for tests on typescript)